src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -40,6 +40,7 @@
     private static final String SEARCH_API_VERSION = "v3";
     public static final String API_PATH = "/api/" + MANAGEMENT_API_VERSION;
     public static final String USERS_API_PATH = API_PATH + "/users";
+    public static final int DEFAULT_PER_PAGE = 10;
```

Comment:
Suggestion: rename to `DEFAULT_ITEMS_PER_PAGE`.

Reply:
Agreed, I think this rename is OK.